ui/src/features/layer/info/pickers/layer-info-picker-for-3dtiles.ts

```diff
@@ -158,10 +191,27 @@ class LayerInfoFor3dTile implements LayerInfo {
     this.title = data.title;
     this.attributes = data.attributes;
 
-    this.originalColor = Color.clone(this.feature.color);
-    this.feature.color = OBJECT_HIGHLIGHT_COLOR.withAlpha(
-      this.feature.color.alpha,
-    );
+    if (this.feature instanceof Cesium3DTileFeature) {
+      this.originalColor = Color.clone(this.feature.color);
+      this.feature.color = OBJECT_HIGHLIGHT_COLOR.withAlpha(
+        this.feature.color.alpha,
+      );
+    } else {
+      const alpha = (this.feature.style?.color as Expression).expression
+        .split(', ')
+        .at(-1)
+        ?.slice(0, -1);
+      this.originalColor = new Color(
+        this.feature.customProperties.color_red as number,
+        this.feature.customProperties.color_green as number,
+        this.feature.customProperties.color_blue as number,
+        alpha as unknown as number,
+      );
+      const color = `color("rgb(${OBJECT_HIGHLIGHT_COLOR.red * 255}, ${OBJECT_HIGHLIGHT_COLOR.green * 255}, ${OBJECT_HIGHLIGHT_COLOR.blue * 255})", 1)`;
```

Comment:
This can be written a tad cleaner with `vec4`:
```suggestion
      const color = `vec4(${OBJECT_HIGHLIGHT_COLOR.red}, ${OBJECT_HIGHLIGHT_COLOR.green}, ${OBJECT_HIGHLIGHT_COLOR.blue}), 1)`;
```
I'm assuming that `color` is passed directly to the shader, for which I'd assume `vec4` to be just a bit faster and/or cleaner than having to parse an rgb color string.